specs/common/knative-serving-conformance-testing-workflow.md

```diff
@@ -0,0 +1,36 @@
+# Knative Conformance Workflow
+
+This document covers the workflow steps to gain knative conformance
+
+## Conformance Steps
+1. Run Manual Tests
+2. Open Github PR in knative/specs repo 
+  - Use PR template below
+  - place in the following directory structure
+    - conformance/tests/results/ directory
+    - conformance/test/results/$vendor directory with a metadata file and test results
+    - ex; .../v1.0/2021-12-18-ibm-code-engine/...
+  - Attach Logs from tests
+3. KTC reviews 
+  - Template
+  - logs
+4. KTC provide approval / exception / or non-approval
+  - This can be done with OWNERS files
```

Comment:
This is a mechanism; I don't think we need to cover it in this document for users.

Reply:
```suggestion
```